reports/performance/_stats.qmd

```diff
@@ -266,27 +266,27 @@ For the empirical cumulative distributions, we are looking for overlap/similar
 distributions. Numeric variables scaled and centered.
 
 ::: panel-tabset
-## Numeric columns:
+## Numeric Columns:
 
 ```{r _stats_ecdfs1}
-grobs <-
-  lapply(names(sf_parcels %>% select(where(is.numeric))), function(x) {
-    data <- sf_parcels %>% select(Sale, any_of(x))
-    ggplot(data, aes(scale(sf_parcels %>% pull(x)),
-      color = Sale
-    )) +
-      stat_ecdf(geom = "step") +
-      labs(x = str_wrap(x, width = 25)) +
-      theme_minimal() +
-      theme(
-        axis.title.y = element_blank(),
-        axis.text.x = element_blank(),
-        axis.ticks.x = element_blank(),
-        axis.text.y = element_blank(),
-        axis.ticks.y = element_blank(),
-        legend.position = "none"
-      )
-  })
+num_plots <- length(sf_parcels %>% select(where(is.numeric)))
```

Comment:
This created a clause that only produces the legend for the final plot in the chart. That means the final plot is slightly smaller.